gradle/shared-with-buildSrc/mirrors.settings.gradle.kts

```diff
@@ -24,6 +24,7 @@ import org.gradle.tooling.events.OperationCompletionListener
 
 class Helper(private val providers: ProviderFactory) {
     val originalUrls: Map<String, String> = mapOf(
+        "jcenter" to "https://jcenter.bintray.com/",
```

Comment:
🤔 why do we want to preserve this?

Reply:
Thanks for pointing this out, we don't need this for release/master